static/src/javascripts/projects/commercial/modules/prebid/utils.js

```diff
@@ -69,7 +69,8 @@ export const getRandomIntInclusive = (
     return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
-export const shouldIncludeAppNexus = (): boolean => isInAuRegion();
+export const shouldIncludeAppNexus = (): boolean =>
+    isInAuRegion() || !isInUsRegion();
```

Comment:
could this be shortened? since `isInAuRegion()` is true when `!isInUsRegion()` is true? Or does this have a benefit for testing? 🤔 hmmm

```
export const shouldIncludeAppNexus = (): boolean => !isInUsRegion();
```